eliot-service/eliot/app.py

```diff
@@ -254,10 +266,13 @@ def get_app(config_manager=None):
         host_id=app_config("host_id"),
         processname="webapp",
     )
-    setup_sentry(
-        basedir=str(REPOROOT_DIR),
-        host_id=app_config("host_id"),
+
+    set_up_sentry(
         sentry_dsn=app_config("secret_sentry_dsn"),
+        release=get_release_name(REPOROOT_DIR),
+        host_id=app_config("host_id"),
+        integrations=[FalconIntegration()],
```

Comment:
There are some issues in the sentry_sdk issue tracker about how the `FalconIntegration` is broken. However, we can see with our integration test that it's definitely doing something, so it seems fine.